playbooks/x-portals-ensure-latest-packages.yml

```diff
@@ -0,0 +1,66 @@
+- name: Ensure Latest Apt Packages Are Installed
+  hosts: webportals
+  remote_user: "{{ webportal_user }}"
+  gather_facts: True
+
+  # Limit concurrency
+  serial: 1
+
+  # Stop on first error, do not execute on the next host
+  any_errors_fatal: True
+
+  # Playbook specific vars
+  vars:
+    max_hosts: "{{ groups['webportals'] | length - 1 }}"
+    docker_compose_build: False
+    set_portal_versions: False
+    portal_action: "portal-ensure-latest-packages"
+    lastpass_required: True
+
+  tasks:
+    - name: Fail if you are targeting all dev and prod webportals
+      include_tasks: tasks/host-limit-check.yml
+
+    - name: Include disabling health check and stopping portal docker services
+      include_tasks: tasks/portal-stop.yml
+
+    - name: Perform apt-get dist-upgrade to update all apt packages
+      ansible.builtin.apt:
+        upgrade: dist
+        update_cache: True
+      become: True
+
+    - block:
```

Comment:
Does having this block here leave the portal completely down? skyd would have been stopped in `tasks/portal-stop.yml` and nothing has brought it back up yet.

Probably should bring it all up, do this tests, and then take it down. This would ensure that the portal continues to be fully functioning and we don't miss updates when we eventually bring it back up.